tests/components/smartthings/test_fan.py

```diff
@@ -1,433 +1,26 @@
-        status={
-            Attribute.switch: "on",
-            Attribute.fan_mode: "high",
-            Attribute.supported_ac_fan_modes: ["high", "low", "medium"],
-        },
-    )
-    await setup_platform(hass, FAN_DOMAIN, devices=[device])
-    # Act
-    await hass.services.async_call(
-        "fan", "turn_off", {"entity_id": "fan.fan_1"}, blocking=True
-    )
-    # Assert
-    state = hass.states.get("fan.fan_1")
-    assert state is not None
-    assert state.state == "off"
-    assert state.attributes[ATTR_PRESET_MODE] == "high"
-
-
-async def test_turn_on_mode_capability(hass: HomeAssistant, device_factory) -> None:
-    """Test the fan turns of successfully."""
-    # Arrange
-    device = device_factory(
-        "Fan 1",
-        capabilities=[Capability.switch, Capability.air_conditioner_fan_mode],
-        status={
-            Attribute.switch: "off",
-            Attribute.fan_mode: "high",
-            Attribute.supported_ac_fan_modes: ["high", "low", "medium"],
-        },
-    )
-    await setup_platform(hass, FAN_DOMAIN, devices=[device])
-    # Act
-    await hass.services.async_call(
-        "fan", "turn_on", {ATTR_ENTITY_ID: "fan.fan_1"}, blocking=True
-    )
-    # Assert
-    state = hass.states.get("fan.fan_1")
-    assert state is not None
-    assert state.state == "on"
-    assert state.attributes[ATTR_PRESET_MODE] == "high"
-
-
-async def test_update_from_signal_mode_capability(
-    hass: HomeAssistant, device_factory
-) -> None:
-    """Test the fan updates when receiving a signal."""
-    # Arrange
-    device = device_factory(
-        "Fan 1",
-        capabilities=[Capability.switch, Capability.air_conditioner_fan_mode],
-        status={
-            Attribute.switch: "off",
-            Attribute.fan_mode: "high",
-            Attribute.supported_ac_fan_modes: ["high", "low", "medium"],
-        },
-    )
-    await setup_platform(hass, FAN_DOMAIN, devices=[device])
-    await device.switch_on(True)
-    # Act
-    async_dispatcher_send(hass, SIGNAL_SMARTTHINGS_UPDATE, [device.device_id])
-    # Assert
-    await hass.async_block_till_done()
-    state = hass.states.get("fan.fan_1")
-    assert state is not None
-    assert state.state == "on"
-
-
-async def test_set_preset_mode_mode_capability(
-    hass: HomeAssistant, device_factory
 ) -> None:
-    """Test setting to specific fan mode."""
-    # Arrange
-    device = device_factory(
-        "Fan 1",
-        capabilities=[Capability.switch, Capability.air_conditioner_fan_mode],
-        status={
-            Attribute.switch: "off",
-            Attribute.fan_mode: "high",
-            Attribute.supported_ac_fan_modes: ["high", "low", "medium"],
-        },
-    )
+    """Test all entities."""
+    await setup_integration(hass, mock_config_entry)
 
-    await setup_platform(hass, FAN_DOMAIN, devices=[device])
-    # Act
-    await hass.services.async_call(
-        "fan",
-        "set_preset_mode",
```

Comment:
The old tests did test interacting with the entity by doing service calls. Why did you remove these tests?
This applies to all platforms and not only to this one